README.md

```diff
@@ -92,24 +92,28 @@ Primazactl help is organized in a hierarchy with contextual help available for d
 ## Command Summary
 
 - Create tenant
+  - checks user has the permissions required to run the command.
```

Comment:
What happens if permissions are not met? Does it provide the appropriate message on what permissions are missing?

Reply:
Added note: 
```
- Create tenant
  - checks user has the permissions required to run the command.
    - if not, create tenant is not performed, and a message is output with details of missing permissions.
```

did similar for other commands.